pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -528,6 +529,35 @@ public void genesisPathDisabledUnderDocker() {
     assertThat(commandOutput.toString()).isEmpty();
   }
 
+  @Test
```

Comment:
Do we need a test to verify that the default behaviour when `--p2p-enabled` isn't used is p2p-enabled = true?

Reply:
Being implicitly tested now as part of the Runner tests.